src/Controller/DefaultController.php

```diff
@@ -143,46 +153,55 @@ public function islandora_view_object(AbstractObject $object) {
     }
 
     arsort($output);
-    \Drupal::moduleHandler()->alter($hooks, $object, $output);
+    $this->moduleHandler()->alter($hooks, $object, $output);
     return $output;
   }
 
   /**
    * Access callback for printing an object.
    */
-  public function islandora_print_object_access($op, $object, AccountInterface $account) {
+  public function islandoraPrintObjectAccess($op, $object, AccountInterface $account) {
     $object = islandora_object_load($object);
     return AccessResult::allowedIf(islandora_print_object_access($op, $object, $account));
   }
 
-  public static function islandora_printer_object(AbstractObject $object) {
+  /**
+   * Islandora printer object.
+   */
+  public static function islandoraPrinterObject(AbstractObject $object) {
     $output = [];
     $temp_arr = [];
 
     // Dispatch print hook.
     foreach (islandora_build_hook_list(ISLANDORA_PRINT_HOOK, $object->models) as $hook) {
-      $temp = \Drupal::moduleHandler()->invokeAll($hook, [$object]);
+      $temp = $this->moduleHandler()->invokeAll($hook, [$object]);
       if (!empty($temp)) {
         $temp_arr = array_merge_recursive($temp_arr, $temp);
       }
     }
-    $output = islandora_default_islandora_printer_object($object, \Drupal::service("renderer")->render($temp_arr));
+    $output = islandora_default_islandora_printer_object($object, $this->renderer->render($temp_arr));
     arsort($output);
 
     // Prompt to print.
     $output['#attached']['library'][] = 'islandora/islandora-print-js';
     return $output;
   }
 
-  public function islandora_object_access($op, $object, $user = NULL, AccountInterface $account) {
+  /**
+   * Islandora object access.
+   */
+  // @codingStandardsIgnoreStart
+  // XXX:params with defaults should be at the end. Risky to move atm.
+  public function islandoraObjectAccess($op, $object, $user = NULL, AccountInterface $account) {
```

Comment:
Looks... like the signature here is actually somewhat broken... Pretty sure `$account` should have _replaced_ the `$user` variable, when it was introduced...

Also... this method doesn't appear to be referenced. Could just chop out cruft?